_docs/setup/install-sidecar.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Integrating Services into the Mesh
+title: Installing Istio Sidecar
```

Comment:
maybe add the redirect from it's previous location for that one too, it's likely to be linked in